README.md

```diff
@@ -59,18 +59,21 @@ pip install pyro-ppl[extras]  # for running examples/tutorials
 
 For recent features you can install Pyro from source.
 
+First, build PyTorch following instructions in the PyTorch
+[README](https://github.com/pytorch/pytorch/blob/master/README.md).
 ```sh
 git clone --recursive https://github.com/pytorch/pytorch
 cd pytorch
 git checkout e40425f  # <---- a well-tested commit
 ```
-Then build PyTorch following instructions in the PyTorch
-[README](https://github.com/pytorch/pytorch/blob/master/README.md).
-Depending on your system, this may be as simple as
+On Linux:
 ```sh
-pip install ninja
 python setup.py install
 ```
+On OSX:
+```sh
+MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
```

Comment:
Thanks for the fix! I somehow missed this, and assumed a linux install. :)

Reply:
You might still need `ninja` though for python 3.6 install (if I remember correctly).